docs/contributing/how-to-open-a-pull-request.md

```diff
@@ -72,28 +72,28 @@ O site da documentação do Gatsby.js está principalmente nos diretórios [www]
 
 Passos adicionais para documentar PR:
 
-- Para alteraçÕes docs-only, considere usar `git checkout -b docs/some-change` ou `git checkout -b docs-some-change`, pois isso irá encurtar o circuito do processo de CI e executar apenas as tarefas de linting.
+- Para alterações que envolvam apenas documentação, considere usar `git checkout -b docs/some-change` ou `git checkout -b docs-some-change`, para encurtar o processo de CI e executar apenas as tarefas de linting.
 
 Mais instruções podem ser encontradas na página de [contribuições de documentações](/contributing/docs-contributions/).
 
 ### Alterações de código
 
 Instruções para fazer alterações no código-fonte do Gatsby, testes, componentes internos, APIs, pacotes e muito mais podem ser encontradas nas documentações de contribuições sobre [configurando seu ambiente de desenvolvimento local](/contributing/setting-up-your-local-dev-environment/). Também há detalhes adicionais na página [Contribuições de código](/contributing/code-contributions/).
 
-### Starters ou Site Showcase
+### Starters ou Amostras de Sites
 
 Existem páginas específicas sobre como contribuir para várias partes do ecossistema de Gatsby:
 
-- [Submissões ao Showcase](/contributing/site-showcase-submissions/)
+- [Submissões de amostras de sites](/contributing/site-showcase-submissions/)
 - [Biblioteca Starter](/contributing/submit-to-starter-library/)
 
-### Blog posts
+### Postagens do Blog
 
-Para o blog do Gatsby, é necessário executar sua ideia de conteúdo pela equipe do Gatsby antes de enviá-la. Para obter mais informações, consulte a página de [contribuições de blog e site](/contributing/blog-and-website-contributions/), incluindo como propor uma idéia e configurar o blog para ser executado localmente.
+Para o blog do Gatsby, é necessário que sua ideia de conteúdo seja aprovada pela equipe do Gatsby antes de ser enviada. Para obter mais informações, consulte a página de [contribuições de blog e site](/contributing/blog-and-website-contributions/), incluindo como propor uma ideia e configurar o blog para ser executado localmente.
 
 ## Atualize seu fork com as alterações mais recentes do Gatsby
 
-O repositório do Gatsby GitHub é muito ativo, portanto, é provável que você precise atualizar seu fork com as alterações mais recentes para poder mergir no seu código. Isso requer adicionar o Gatsby como um [upstream remote](https://help.github.com/pt/articles/configuring-a-remote-for-a-fork):
+O repositório do Gatsby GitHub é muito ativo, portanto, é provável que você precise atualizar seu fork com as alterações mais recentes para poder submeter no seu código. Isso requer adicionar o Gatsby como um [upstream remote](https://help.github.com/pt/articles/configuring-a-remote-for-a-fork):
```

Comment:
1) "submeter no seu código" ... isso ta correto?
2) "This requires" -> "Isso requer" , nao aconselho traduzir usando a mesma estrutura do ingles porque comecar a frase com esse pronome nao soa bem em portugues. Se tiver outra solucao aqui seria ótimo.

Reply:
Não acho que "isso requer" esteja estranho